backend/app/routers/authentication.py

```diff
@@ -201,3 +201,67 @@ async def revoke_admin(
             status_code=403,
             detail=f"User {current_username.email} is not an admin. Only admin can revoke admin access.",
         )
+
+
+@router.post("/users/enable/{useremail}", response_model=UserOut)
+async def user_enable(
+    useremail: str, current_username=Depends(get_current_user), admin=Depends(get_admin)
+):
+    if admin:
+        if current_username.email == useremail:
+            raise HTTPException(
+                status_code=403,
+                detail=f"You are currently an admin. Admin cannot enable their own self.",
+            )
+        else:
+            if (user := await UserDB.find_one(UserDB.email == useremail)) is not None:
+                try:
+                    await enable_disable_user(useremail, True)
+                except KeycloakGetError as e:
+                    raise HTTPException(
+                        status_code=e.response_code,
+                        detail=json.loads(e.error_message),
+                        headers={"WWW-Authenticate": "Bearer"},
+                    )
+                return user.dict()
+            else:
+                raise HTTPException(
+                    status_code=404, detail=f"User {useremail} not found"
+                )
+    else:
+        raise HTTPException(
+            status_code=403,
+            detail=f"User {current_username.email} is not an admin. Only admin can enable user access.",
+        )
+
+
+@router.post("/users/disable/{useremail}", response_model=UserOut)
+async def user_enable(
```

Comment:
Do you mind renaming this function to user_disable? Same function names might be confusing and this function is actually disabling users. 
 Rest look all good to me.

Reply:
fixed it. Thank you.